libraries/AP_RCProtocol/AP_RCProtocol_SUMD.cpp

```diff
@@ -113,26 +90,13 @@ void AP_RCProtocol_SUMD::_process_byte(uint32_t timestamp_us, uint8_t byte)
         break;
 
     case SUMD_DECODE_STATE_GOT_HEADER:
-        if (byte == SUMD_ID_SUMD || byte == SUMD_ID_SUMH) {
+        if ((byte == SUMD_ID_SUMD) || (byte == SUMD_ID_FAILSAFE)) {
```

Comment:
This is more than just removing legacy cruft, which is what the PR is claiming...

Reply:
True, I forgot to mention this, but without this change we just stop seeing input when the receiver reports itself as in failsafe, which then means you wait for a timeout. In practice the actual docs for the SUMD protocol I could find seem to indicate this flag is only really set in some weird conditions. I'm fine with removing it, but if I remove it then I should also remove the call for `add_input()` on it.

Are you happy with the change? Definitely my bad for not calling it out.